widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadLayout.kt

```diff
@@ -39,6 +39,7 @@ internal class ChatHeadLayout @JvmOverloads constructor(
 
     private val chatHeadSize: Float by lazy { resources.getDimension(R.dimen.glia_chat_head_size) }
     private val chatHeadMargin: Float by lazy { resources.getDimension(R.dimen.glia_chat_head_content_padding) }
+    private val bubbleDragSpaceMargin by lazy { resources.getDimension(R.dimen.glia_small) }
```

Comment:
NIT NIT. Maybe just `bubbleMargin`? 🙂 
'Space' duplicates the 'Margin', per se. 'Drag' doesn't seem to be an important detail. 🙂

Reply:
`chatHeadMargin` is about the same, but only applied to the right and bottom borders(when the bubble is initially placed and when dragging). The `bubbleDragSpaceMargin` is applied to the left and top borders only when user moves the bubble.
I was unable to use the initial `chatHeadMargin` because, the bubble itself has some asymmetry in internal paddings.
Maybe just name them `chatHeadBottomRightMargin`, `chatHeadTopRight` margin?

Reply:
Oh, I didn't notice `chatHeadMargin`. Yes, I like the idea of `chatHeadBottomRightMargin`, `chatHeadTopRight`. It reflects the actual meaning and is clear.

Reply:
Changed